src/components/Forms/tcomb/textbox.js

```diff
@@ -33,7 +33,7 @@ function textbox(locals) {
           borderBottomWidth: 0.5,
         }}
       >
-        <View style={{ width: 25 }}>
+        <View style={{ width: 25, justifyContent: 'center', alignItems: 'center' }}>
```

Comment:
prefer all styles to be using a StyleSheet. This is a `View` so it should play well with `StyleSheet`s, any time you use a component from `native-base` you'll have to do `StyleSheet.flatten(styles.styleName)`